component/prometheus/exporter/mssql/mssql.go

```diff
@@ -40,6 +41,7 @@ type Arguments struct {
 	MaxIdleConnections int               `river:"max_idle_connections,attr,optional"`
 	MaxOpenConnections int               `river:"max_open_connections,attr,optional"`
 	Timeout            time.Duration     `river:"timeout,attr,optional"`
+	QueryConfigPath    string            `river:"query_config_path,attr,optional"`
```

Comment:
The more river centric path for this would for this to be query_config and allow them to use local.file to get the content. That way they could get it in a myriad number of ways or inline if they really really wanted to.